### PR TITLE
Rule: Remote Access Software

### PR DESCRIPTION
**Pull request by ProtoElectron**
Created two rules for detecting remote access software (service creation and domain name requests.) Did this by testing about 20 of them.

### CODE REVIEW

File: rules/windows/builtin/security/win_remote_access_software_installed.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        - ServiceFileName|contains: 'GoToAssist'
+        - ServiceFileName|contains: 'AmmyyAdmin'
+        - ServiceFileName|contains: 'vncserver'
+        - ServiceFileName|contains: 'Parsec'
+        - ServiceFileName|contains: 'chromoting'
+        - ServiceFileName|contains: 'Zoho'
+        - ServiceFileName|contains: 'jumpcloud'
+        - ServiceFileName|contains: 'kaseya'
+        
+    condition: selection and selection_1
```
Comment (frack113) [inline rules/windows/builtin/security/win_remote_access_software_installed.yml:70]: Or 
```
    selection:
        EventID: 
            - 4697
            - 7045
        ServiceFileName|contains: 
            - 'AnyDesk'
            - 'SSUService'
            - 'SplashtopRemoteService'
       [...]
            -  'kaseya'
    condition: selection 
```

File: rules/windows/builtin/security/win_remote_access_software_installed.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+author: Connor Martin
+date: 2022/03/13
+logsource:
+    product: windows
+    service: security
+detection:
+    selection:
+        EventID: 
+            - 4697
+            - 7045
```
Comment (frack113) [inline rules/windows/builtin/security/win_remote_access_software_installed.yml:45]: 4697 is `service: security` ✔️ 
7045 is `service: system` so will never works

File: rules/windows/dns_query/dns_query_win_remote_access_software.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    - https://redcanary.com/blog/misbehaving-rats/
+author: Connor Martin
+date: 2022/03/13
+logsource:
+    product: windows
+    category: dns_query
+detection:
+    selection:
+        QueryName|endswith: 
+        - '.net.anydesk.com'
```
Comment (frack113) [inline rules/windows/dns_query/dns_query_win_remote_access_software.yml:42]: Can you use indentation for human understanding like
```
detection:
    selection:
        QueryName|endswith: 
            - '.net.anydesk.com'  
```

Thanks

**Review by frack113 — COMMENTED** [review]
Hi,
I just check the logic .
Are you sure some are not allready cover ?

**Comment by Neo23x0** [comment]
Problem: 
Events with ID 4697 in Security use "Service File Name" 
Events with ID 7045 in System use "ImagePath" 

Don't look at the localised version in the "General" Tab but the Field Name in the "Details" Tab. 

Screenshot 2022-03-22 at 13 20 00
Screenshot 2022-03-22 at 13 20 21

**Comment by nasbench** [comment]
Closed this due to inactivity and moved the content to #3816

**Pull request closed without merging**